### PR TITLE
OCTOPUS-622: image download and create task

### PR DESCRIPTION
**Pull request by pkenchap**
This PR to download the image and create/upload the image from the location .

### CODE REVIEW

File: openstack/intel-worker/tasks/ignition.yml [hunk 1]
```diff
@@ -28,3 +28,24 @@
     template:
       src: ../templates/worker-amd64.ign.j2
       dest: /tmp/worker-amd64.ign
+
+  - name: "Download the Image from the URL and use to create image"
+    ansible.builtin.get_url:
+      url: "{{ image_url }}"
+      dest: "{{ image_dest_filename  }}"
+      mode: '0755'
+      validate_certs: false
```
Comment (prb112) [inline openstack/intel-worker/tasks/ignition.yml:37]:
```suggestion
      validate_certs: false
```
Let's have the validate_certs come be a var

File: openstack/intel-worker/tasks/ignition.yml [hunk 2]
```diff
@@ -28,3 +28,24 @@
...
+      cloud: openstack
+      state: present
+      name: "{{ image_name }}"
+      container_format: "{{ image_container_format }}"
+      disk_format: "{{ image_disk_format }}"
+      filename: "{{ image_dest_filename }}"
+      is_public: "{{ image_is_public }}"
+      properties:
+        cpu_arch: "{{ image_cpu_arch }}"
+        distro: "{{ image_distro }}"
```
[the remark this answers is not in the document]
Comment (prb112) [reply openstack/intel-worker/tasks/ignition.yml:51]: Please consult the original issue

**Comment by prb112** [comment]
Let's use `OCTOPUS-622: image download and create task` as this follows a best practice.

File: openstack/intel-worker/tasks/ignition.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (pkenchap) [reply openstack/intel-worker/tasks/ignition.yml]: Sure , added different files for create and remove image files . 
Thanks

File: openstack/intel-worker/tasks/ignition.yml (hunk 1 above)
Comment (pkenchap) [reply openstack/intel-worker/tasks/ignition.yml]: Sure added to vars , thanks

File: openstack/intel-worker/vars/main.yml [hunk 3]
```diff
@@ -17,3 +17,13 @@ dns_none_conf: |-
 # Used in patch nfs deployment task
 nfs_namespace: "nfs-provisioner"
 nfs_deployment: "nfs-client-provisioner"
+
+# Image details
+image_name: "centos7_pun_img"
+image_container_format: "bare"
+image_disk_format: "qcow2/raw"
+image_dest_filename: "/tmp/CentOS-7-x86_64-GenericCloud-1503.qcow2 /tmp/rhcos-openstack.x86_64.qcow2.gz"
```
[the remark this answers is not in the document]
Comment (pkenchap) [reply openstack/intel-worker/vars/main.yml]: It was just a example to have a same var files , added to different vars for rhcos and centos . Thanks .

**Comment by pkenchap** [comment]
Updated with the Create and remove image for centos and rhcos yml files .  
@prb112 Please review . 

Thanks .

**Review by prb112 — APPROVED** [review]
/LGTM

**Pull request merged**